rojak-pantau/rojak_pantau/spiders/metrotvnews.py

```diff
@@ -0,0 +1,190 @@
+
+        if ROJAK_SLACK_TOKEN != '':
+            self.is_slack = True
+            self.slack = Slacker(ROJAK_SLACK_TOKEN)
+        else:
+            self.is_slack = False
+            self.logger.info('Post error to #rojak-pantau-errors is disabled')
+
+    # Capture the signal spider_opened and spider_closed
+    # https://doc.scrapy.org/en/latest/topics/signals.html
+    @classmethod
+    def from_crawler(cls, crawler, *args, **kwargs):
+        spider = super(MetrotvnewsSpider, cls).from_crawler(crawler,
+                *args, **kwargs)
+        crawler.signals.connect(spider.spider_opened,
+                signal=signals.spider_opened)
+        crawler.signals.connect(spider.spider_closed,
+                signal=signals.spider_closed)
+        return spider
+
+    def spider_opened(self, spider):
+        # Using UTF-8 Encoding
+        self.db.set_character_set('utf8')
+        self.cursor.execute('SET NAMES utf8;')
+        self.cursor.execute('SET CHARACTER SET utf8;')
+        self.cursor.execute('SET character_set_connection=utf8;')
+
+    def spider_closed(self, spider, reason):
+        spider.logger.info('Spider closed: {} {}'.format(spider.name, reason))
+
+        # update last_scraped_at if spider finished without error
+        if reason == 'finished':
+            try:
+                self.logger.info('Updating media last_scraped_at information')
+                self.cursor.execute(sql_update_media, [datetime.now(),
+                    self.name])
+                self.db.commit()
+                self.db.close()
+            except mysql.Error as err:
+                self.logger.error('Unable to update last_scraped_at: {}'.format(err))
+                self.db.rollback()
+                self.db.close()
+
+        else:
+            if self.is_slack:
+                # Send error to slack
+                error_msg = '{}: Unable to update last_scraped_at: {}'.format(
+                    spider.name, err)
+                self.slack.chat.post_message('#rojak-pantau-errors',
+                    error_msg, as_user=True)
+
+    def parse(self, response):
+        self.logger.info('parse: {}'.format(response))
+        is_scraped = False
+
+        # Collect list of news from current page
+        for i, article in enumerate(
+                response.css('div.topic') + response.css('li:not(.last) > div.grid')):
+            if i == 0:
+                url = article.css('h1 > a::attr(href)').extract()[0]
+            else:
+                url = article.css('h2 > a::attr(href)').extract()[0]
+
+            # Example: Minggu, 09 Oct 2016 15:14
+            info = article.css('div.reg::text').extract()[1].strip()
+
+            # Parse date information
+            try:
+                # Example: 09 Oct 2016 15:14
+                info_time = info.split(',')[1].strip()
+                self.logger.info('info_time: {}'.format(info_time))
+                published_at = datetime.strptime(info_time,
```

Comment:
Fixed in https://github.com/pyk/rojak/pull/48/commits/cacb9c8041216a1470ee7f9af0f4ab2fff7f34ff